video/out/wayland_common.c

```diff
@@ -1106,13 +1110,13 @@ static void registry_handle_add(void *data, struct wl_registry *reg, uint32_t id
         wl->dmabuf = wl_registry_bind (reg, id, &zwp_linux_dmabuf_v1_interface, 2);
         zwp_linux_dmabuf_v1_add_listener (wl->dmabuf, &dmabuf_listener, wl);
         wl->drm_format_ct_max = 64;
-        wl->drm_formats = talloc_array(NULL,uint,wl->drm_format_ct_max);
+        wl->drm_formats = talloc_array(wl, uint32_t, wl->drm_format_ct_max);
     }
 
     if (!strcmp (interface, wp_viewporter_interface.name) && (ver >= 1) && found++) {
        wl->viewporter = wl_registry_bind (reg, id, &wp_viewporter_interface, 1);
        wl->viewport = wp_viewporter_get_viewport (wl->viewporter, wl->surface);
```

Comment:
These two have spaces too.

Reply:
`wl->viewporter = wl_registry_bind (reg, id, &wp_viewporter_interface, 1);` and `wl->viewport = wp_viewporter_get_viewport (wl->viewporter, wl->surface);` still have a space before the parenthesis.